pyproject.toml

```diff
@@ -93,6 +93,7 @@ multi_line_output = 3
 include_trailing_comma = true
 force_grid_wrap = 0
 use_parentheses = true
+src_paths = ["src/py", "examples"]
```

Comment:
Why do we need to configure this when we pass the directories `isort` should check in `test.sh` etc.?

Reply:
We actually don't, I will remove this line from the `pyproject.toml` .